pkg/search/proxy/store/util_test.go

```diff
@@ -212,6 +212,41 @@ func Test_newMultiClusterResourceVersionFromString(t *testing.T) {
 				},
 			},
 		},
+		{
+			name: "success - empty cluster name",
```

Comment:
Just a question: In real production, will clusterName be empty?

Reply:
No，just to test this special case

Reply:
This is request from client by users. We cannot assume the request is always valid. So we have to have solutions to handle the invalid request.

Reply:
Ok, thanks~